### PR TITLE
[EA Forum only] link to "Take Action" tag from the sidebar

### PR DESCRIPTION
**Pull request by s-cheng**
The EA Forum (i.e. Max and I) will be exploring how we can better facilitate people taking significant actions. We are adding this link to our sidebar as a quick way to get some information on what opportunities people are interested in.

Screen Shot 2022-02-28 at 11 56 32 AM

### CODE REVIEW

File: packages/lesswrong/components/events/modules/EventCards.tsx [hunk 1]
```diff
@@ -185,12 +185,12 @@ const EventCards = ({events, loading, numDefaultCards, hideSpecialCards, classes
...
       // for logged in users, just display the In-Depth / Precipice VP card
-      eventCards.splice(2, 0, <VirtualProgramCard program="advanced" />)
+      eventCards.splice(2, 0, <VirtualProgramCard key="advancedVP" program="advanced" />)
     } else {
       // for logged logged out users, display both VP cards
-      eventCards.splice(2, 0, <VirtualProgramCard program="intro" />)
+      eventCards.splice(2, 0, <VirtualProgramCard key="introVP" program="intro" />)
       // we try to space out the two cards
-      eventCards.splice(5, 0, <VirtualProgramCard program="advanced" />)
+      eventCards.splice(5, 0, <VirtualProgramCard key="advancedVP" program="advanced" />)
```
Comment (s-cheng) [inline packages/lesswrong/components/events/modules/EventCards.tsx:193]: Also fixed an issue here

**Comment by jpaddison3** [comment]
Ok as temporary. Longer term we'd need to figure out how to better chunk up the sidebar, let people's eyes stop being able to scan it.

File: packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx [hunk 2]
```diff
@@ -33,8 +33,14 @@ const TabNavigationMenu = ({onClickSection, classes}: {
   classes: ClassesType,
 }) => {
   const currentUser = useCurrentUser();
+  const { captureEvent } = useTracking()
   const { TabNavigationItem, FeaturedResourceBanner } = Components
   const customComponentProps = {currentUser}
+  
+  const handleClick = (e, tabId) => {
+    captureEvent(`${tabId}NavClicked`)
```
Comment (Xodarap) [inline packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx:41]: I only see a linkClicked event, not this one you added. I'm just clicking on the sidebar menu. Not sure if I'm doing something wrong?

image

**Review by Xodarap — APPROVED** [review]
I can't get the analytics thing you added to work, but I could be doing something wrong. Otherwise this looks good :shipit:

File: packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx (hunk 2 above)
Comment (s-cheng) [reply packages/lesswrong/components/common/TabNavigationMenu/TabNavigationMenu.tsx:41]: Hmm I saw it in the console when I tested - I'll go ahead and deploy this, and if it's not working on prod I'll look into it more

**Pull request merged**